WTWSMS/localisation/WtWSMS_religions.csv

```diff
@@ -421,15 +401,8 @@ job_chancellor_celtic_pagan;Seannsalair;;;;;;;;;;;x
 job_spymaster_celtic_pagan;Brath;;;;;;;;;;;x
 job_treasurer_celtic_pagan;Rianadair;;;;;;;;;;;x
 job_marshal_celtic_pagan;Curaidh;;;;;;;;;;;x
-job_spiritual_celtic_pagan;Court Druid;;;;;;;;;;;x
+#job_spiritual_celtic_pagan;Court Druid;;;;;;;;;;;x
```

Comment:
Use ACR localization to avoid confusion, as Court Druid is a minor title required for some decisions.